install.sh

```diff
@@ -33,6 +33,30 @@ MAGENTA='\e[1;35m';
 RESET='\e[0m'
 
 echo -e "${GREEN}Welcome to Chromebrew!${RESET}\n"
+
+# load system info
+if [ -f /etc/lsb-release ]; then
+  source /etc/lsb-release
+else
+  echo -e "${YELLOW}Unable to detect system information, installation will continue.${RESET}"
+  CHROMEOS_RELEASE_VERSION=0.0-00000
+fi
+
+# disallow non Chrome OS devices
+if [ -z "${CHROMEOS_RELEASE_VERSION}" -a "${CREW_FORCE_INSTALL}" != '1' ]; then
```

Comment:
This will never be true, correct?

Reply:
It means if both CHROMEOS_RELEASE (check if it is chrome os) and FORCE_INSTALL (skip switch) not set then exit

Reply:
The logic above always sets `CHROMEOS_RELEASE_VERSION`, right?

Reply:
The script only check existent of crosh now